### PR TITLE
Add missing README entry for es:migrate command

### PR DESCRIPTION
**Pull request by alexandrebouthinon**
## What does this PR do?

Add missing README entry for es:migrate command

### How should this be manually tested?

Type `kourou es:migrate --help` and compare with readme entry 

### Boyscout

Fix index:export description in the source code for consistent future README generation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -529,6 +530,30 @@ OPTIONS
 
 _See code: [src/commands/es/indices/insert.ts](src/commands/es/indices/insert.ts)_
 
+## `kourou es:migrate`
+
+Migrate all the index from one Elasticsearch server to another
```
Comment (xbill82) [inline README.md:535]:
```suggestion
Migrate all the indexes from one Elasticsearch server to another
```

File: README.md [hunk 2]
```diff
@@ -529,6 +530,30 @@ OPTIONS
...
+
+```
+USAGE
+  $ kourou es:migrate
+
+OPTIONS
+  --batch-size=batch-size  [default: 1000] How many documents to move in batch per operation
```
Comment (xbill82) [inline README.md:542]:
```suggestion
  --batch-size=batch-size  [default: 1000] The amount of documents to move in batch per operation
```

File: src/commands/index/export.ts [hunk 3]
```diff
@@ -20,7 +20,7 @@ export default class IndexExport extends Kommand {
       default: '2000'
     }),
     format: flags.string({
-      description: '"jsonl or kuzzle - kuzzle will export in Kuzzle format usable for internal fixtures and jsonl will be usable to import that data back with kourou',
+      description: '"jsonl or kuzzle - kuzzle will export in Kuzzle format usable for internal fixtures and jsonl allows to import that data back with kourou',
```
Comment (xbill82) [inline src/commands/index/export.ts:23]:
```suggestion
      description: '"The output format. Allowed values are `jsonl` and `kuzzle`. `kuzzle` will export data in the bulk:import format, while `jsonl` will export data in the format used by Kourou"',
```

**Review by xbill82 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (alexandrebouthinon) [reply README.md:535]: I'll update it in the migrate.ts file and generate a new README

File: README.md (hunk 2 above)
Comment (alexandrebouthinon) [reply README.md:542]: I'll update it in the migrate.ts file and generate a new README

**Comment by Leodau** [comment]
Nice! :muscle:

**Review by Leodau — APPROVED** [review]

**Pull request merged**